File: java/app/src/main/java/hangman/models/Game.java
package hangman.models;

import com.google.gson.annotations.Expose;

import java.util.ArrayList;
import java.util.List;

public class Game {
    @Expose
    private int remainingGuesses;
    @Expose
    private String word;
    @Expose(serialize = false, deserialize = false)
    private String unmaskedWord;
    @Expose
    private List<String> incorrectGuesses;
    @Expose
    private String status;

    public Game(int guesses, String selectedWord) {
        remainingGuesses = guesses;
        unmaskedWord = selectedWord;
        word = selectedWord.replaceAll("[a-zA-Z0-9_]", "_");
        status = "In Progress";
        incorrectGuesses = new ArrayList<>();
    }
}

File: dotnet/src/api/Models/Game.cs
namespace dotnet.Models;

public class Game {
    
    public int RemainingGuesses { get; set; }
    public string Word;
    public string UnmaskedWord { get; set; }
    public List<string> IncorrectGuesses { get; set; }
    public string Status { get; set; }

}